src/lti/contrib/django/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class NonceHistory(models.Model):
+    client_key = models.CharField(max_length=200)
+    timestamp = models.IntegerField()
+    nonce = models.CharField(max_length=200)
```

Comment:
I think that the timestamp probably needs to be a datetime field. An enhancement for this would be to allow nonces to be stored in the cache instead of the database, for the same reasons that we want to do that for sessions. And, like the sessions table, this needs to have a management command to clear out unneeded nonces, or else this table will end up growing unbounded. This caveat also needs to be documented.

Reply:
@ryanhiebert I don't have a strong opinion of int vs datetime here. I'd be inclined to do away with the whole model and delegate nonce storage to whatever cache backend the user has configured. Interested to know your thoughts.

Reply:
That makes sense. In re-reading it, I'd missed that this is for the `NonceHistory` model. Using the cache makes sense to me. I can imagine wanting to make it configurable, but my feeling is that we'll be able to add that if we need it in the future without breaking compatibility.